pandas/core/indexing.py

```diff
@@ -1450,11 +1425,11 @@ class _LocationIndexer(_NDFrameIndexer):
     def __getitem__(self, key):
         if type(key) is tuple:
```

Comment:
should prob use isinstance here as this is an old idiom

Reply:
tried this, turns out this is intentional to get tuple but exclude NamedTuple.  Added a comment to that effect.  (green)